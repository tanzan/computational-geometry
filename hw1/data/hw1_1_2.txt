-4 0 2 -10
8
-8 4
2 -7
9 3
-4 6
-8 -5
7 1
2 4
-3 -9