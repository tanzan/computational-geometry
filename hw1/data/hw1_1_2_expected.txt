RIGHT
LEFT
LEFT
LEFT
RIGHT
LEFT
LEFT
RIGHT